current/hackeve47_intro_to_pygame

```diff
@@ -0,0 +1,24 @@
+---
+type: Hackeve 47
```

Comment:
It should be just "HackEve". 
You can make the name as HackEve 45: Intro to pygame.